lua/keymaps.lua

```diff
@@ -0,0 +1,11 @@
+-- Move current line/s under cursor up or down
+vim.keymap.set("n", "J", ":m .+1<CR>==", { desc = "Move current line down" })
+vim.keymap.set("n", "K", ":m .-2<CR>==", { desc = "Move current line up" })
```

Comment:
Very nice!

This coming from https://vim.fandom.com/wiki/Moving_lines_up_or_down?

We could add the source in a comment, given it has good explanation for the mappings.